packages/grid_client/src/modules/gateway.ts

```diff
@@ -59,10 +89,25 @@ class GWModule extends BaseModule {
     return { contracts: contracts };
   }
 
+  /**
+   * Deploys a gateway with a given name.
+   *
+   * This method deploys a gateway with the provided name configuration.
```

Comment:
please add to the description that this name will be used as a subdomain to the gateway's domain
```suggestion
   * Deploys a gateway workload with a given name.
   *
   * This method deploys a gateway workload with the provided name configuration.
```